src/Azure.Functions.Sdk/TaskItemExtensions.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Build.Framework;
+
+namespace Azure.Functions.Sdk;
+
+public static class TaskItemExtensions
```

Comment:
internal? do we want to do that for all extensions/classes unless intended to be public?